Django_web/myfirst/templates/articles/home_list.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+
+{% block title %}Домашняя страница{% endblock %}
+
+{% block content %}
+
+    <h3><a href="/articles"> Посмотреть список статей</a></h3>
+
+
+{% endblock %}
```

Comment:
To fix these kind of warnings, please add a new line at end of the files

Reply:
Done